packages/frontend/src/common/components/navigation/Header.tsx

```diff
@@ -1,9 +1,46 @@
-import React from 'react'
+'use client'
+import React, { useEffect, useState } from 'react'
 import Link from 'next/link'
 import Image from 'next/image'
 import { NavigationItem } from '@/common/components/navigation/NavigationItem'
+import { getUser, logOut } from '@/api'
+import { usePathname } from 'next/navigation'
+import { useRouter } from 'next/navigation'
+
+interface User {
+  id?: string
+}
 
 export function Header() {
+  const pathname = usePathname()
+  const router = useRouter()
+  const [user, setUser] = useState<User>({})
+  const protectedPages = ['/submissions', '/invitations', '/invitations/new']
+
+  useEffect(() => {
+    getUser().then((res) => {
```

Comment:
Is this response, result or user? :)
```suggestion
    getUser().then((user) => {
```

Reply:
Changed to a `user`